profiler/src/ProfilerEngine/Datadog.Profiler.Native/SamplesCollector.cpp

```diff
@@ -102,17 +102,17 @@ void SamplesCollector::Export()
         // batched samples are collected once just before export
         CollectSamples(_batchedSamplesProviders);
 
-        Log::Debug("Collected samples per provider:");
+        Log::Info("Collected samples per provider:");
```

Comment:
This will increase the customer logs size in an unbound way: better to keep it as it is

Reply:
yep my bad, I reverted it